SUI/SUI/PlayerViewModel.swift

```diff
@@ -0,0 +1,97 @@
+//  PlayerViewModel.swift
+//  SUI
+//  Created by angelina on 19.01.2023.
+
+
+import AVFoundation
+
+/// Модель плеера
+final class PlayerViewModel: ObservableObject {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let songsValues = ["gori", "dymd", "newYear"]
+        static let songTypeString = "mp3"
+        static let timeInterval = 0.1
+        static let stubDouble = 0.0
+        static let songIndex = 0
+        static let zeroNumber = 0
+        static let oneNumber = 1
+    }
+    
+    // MARK: - Public properties
+    
+    @Published var maxDuration = Constants.stubDouble
+    @Published var currentDuration: Double = Constants.stubDouble
+    var player: AVAudioPlayer?
```

Comment:
```suggestion
  private   var player: AVAudioPlayer?
```
мы через метод обращаемся в плееру поэтому private лучше